main_routers/music_router.py

```
@@ -65,6 +68,209 @@ def _patch_pyncm_async() -> None:
+            if content_length:
+                try:
+                    declared_size = int(content_length)
+                    if declared_size > MAX_MUSIC_SIZE:
+                        await resp.aclose()
+                        logger.warning(f"[Music Proxy] 音乐文件过大: {declared_size}")
+                        return JSONResponse(content={"success": False, "error": "音乐文件超过大小限制 (50MB)"}, status_code=413)
+                except (ValueError, TypeError):
+                    pass
+
+            if declared_size >= STREAMING_SIZE_THRESHOLD:
+                # 大文件：关掉探测流，交给 _stream_music 用独立 client 流式传输
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)